contentcuration/contentcuration/tasks_test.py

```diff
@@ -32,6 +32,17 @@ def error_test_task(**kwargs):
     raise Exception("I'm sorry Dave, I'm afraid I can't do that.")
 
 
+@app.task(bind=True, name="caught_error_test_task")
+def caught_error_test_task(self, **kwargs):
+    """
+    This is a mock task designed to test that we properly report errors to the client.
+    """
+    try:
+        raise Exception("I'm sorry Dave, I'm afraid I can't do that.")
```

Comment:
I just got a tiny bit of deja vu - all of the error messages in the original version of KA Lite that we released in December 2012 were Hal 9000 quotes.

Reply:
I can't take credit as I copy-pasted it, but I did enjoy the reference!